WcaOnRails/config/routes.rb

```diff
@@ -51,6 +53,9 @@
   patch 'competitions/:competition_id/report' => 'delegate_reports#update'
 
   get 'competitions/:id/edit/admin' => 'competitions#admin_edit', as: :admin_edit_competition
+  get 'competitions/:id/payment_setup' => 'competitions#payment_setup', as: :competitions_payment_setup
+  post 'competitions/:id/revoke_stripe_access' => 'competitions#revoke_stripe_access', as: :competitions_revoke_stripe_access
+  get 'stripe-connect' => 'competitions#stripe_connect'
```

Comment:
This might need to get a `as: :competitions_stripe_connect` for my other suggestion to work. You should run `bin/rake routes` first to verify that there isn't already a "Prefix" for this.